lib/scenic/adapters/postgres/index_reapplication.rb

```diff
@@ -35,6 +35,18 @@ def on(name)
           indexes.each(&method(:try_index_create))
         end
 
+        def on_side_by_side(name, new_table_name, temporary_id)
+          indexes = Indexes.new(connection: connection).on(name)
+          indexes.each_with_index do |index, i|
+            old_name = "predrop_index_#{temporary_id}_#{i}"
```

Comment:
We should probably _require_ you be inside a ddl_transaction to use side_by_side. It would be pretty bad if the migration failed after renaming indexes on the existing (live) view, right?